ovs-p4rt/ovs_p4rt.cc

```diff
@@ -1477,6 +1482,7 @@ void ConfigFdbTableEntry(struct mac_learning_info learn_info,
       auto status_or_read_response =
           GetFdbTunnelTableEntry(session.get(), learn_info, p4info);
       if (status_or_read_response.ok()) {
+        printf("TUNNEL: read FDB L2_FWD_TX_TABLE entry present\n");
```

Comment:
This looks like a leftover debug statement. It should be removed or replaced with a call to the logger.

Reply:
This was intentionally kept for debugging. Right now ovs-sidecar doesnt have any logging mechanism. This was not available from day 1.